package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "newsAppNode",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "cross-env node src/app.js"
+  },
+  "author": "Liza Ivashkova",
+  "license": "ISC",
+  "devDependencies": {
+    "body-parser": "^1.19.0",
+    "cross-env": "^6.0.3",
+    "express": "^4.17.1",
+    "morgan": "^1.9.1"
+  }
```

Comment:
You need these dependencies to run the server, therefore trey are just dependencies.
devDependencies should contain things required for development, such as ESLint, tests runner, etc.

Reply:
done